src/states_screens/feature_unlocked.cpp

```diff
@@ -271,12 +290,17 @@ void FeatureUnlockedCutScene::addTrophy(RaceManager::Difficulty difficulty)
 }
 
 // ----------------------------------------------------------------------------
-// unused for now, maybe will be useful later?
 
-void FeatureUnlockedCutScene::addUnlockedKart(KartProperties* unlocked_kart,
-                                              irr::core::stringw msg)
+void FeatureUnlockedCutScene::addUnlockedKart(KartProperties* unlocked_kart)
 {
-    assert(unlocked_kart != NULL);
+    if (unlocked_kart == NULL)
+    {
+        Log::error("FeatureUnlockedCutScene::addUnlockedKart", "Unlocked kart does not exist");
+        return;
+    }
+    irr::core::stringw name = unlocked_kart->getName();
+    irr::core::stringw msg = _("You unlocked ");
```

Comment:
Prefer using string formatting such as _("You unlocked %s!", name), this will be much clearer and flexible for translators